src/pump/_resourcepolicy.py

```diff
@@ -32,6 +32,9 @@ def imported(self):
 
     @time_method
     def import_to(self, env, dspace, repo):
+        DEFAULT_BITSTREAM_READ = "DEFAULT_BITSTREAM_READ"
+        ITEM = "ITEM"
```

Comment:
duplicate, use one static variable e.g., collections.ITEM otherwise you change one string and have a sync problem